tools/build_cros_gcc

```diff
@@ -61,23 +64,23 @@
  echo GLIBC=$GLIBC
  shift
  exit 0;;
- armv7l-cros-linux-gnueabihf)
+ xarmv7l-cros-linux-gnueabihfx)
  true;;
- aarch64-cros-linux-gnu)
+ xaarch64-cros-linux-gnux)
  true;;
- i686-cros-linux-gnu)
+ xi686-cros-linux-gnux)
  true;;
- x86_64-cros-linux-gnu)
+ xx86_64-cros-linux-gnux)
  true;;
- *)
+ xx)
+ $0 -h
+ exit 1;;
+ x*x)
  echo Invalid target or option \'$1\'
```

Comment:
Why not red text here also? ;)

Reply:
Because it's not an error that would stop them from moving on totally.

Reply:
But the red text would make it stand out.  BTW, red doesn't necessarily mean a fatal error.

Reply:
OK. It's red now.